src/globals/scss/_vars.scss

```diff
@@ -1 +1,3 @@
 $global-images: "/images/";
+$site-width: 960px;
```

Comment:
I think we should use the `govuk` variable prefix here, can we add it for $global-images too as it is missing.